app/workers/legacy_sync_worker.rb

```diff
@@ -0,0 +1,15 @@
+require 'rest-client'
+
+class LegacySyncWorker
+  include Sidekiq::Worker
+
+  def perform(id)
+    json = Dataset.find(id).ckanify_metadata.to_json
+    server =
+      Rails.env.production? ? "https://data.gov.uk" : "https://test.data.gov.uk"
```

Comment:
The worker is only called if the env is production, but have added as a second line of defence as I was nervous of breaking legacy! Perhaps there's a better safety mechanism...

Reply:
This means running tests requires test.data.gov.uk? That's probably not ideal as it depends on test.dgu being up, but also we cannot run the tests offline. Is mocking a possibility?

Reply:
yeah I think it might be a good idea to mock data.gov.uk in our tests here.